### ISSUE TITLE
[Bug Bash] Hard to see the package list text-box of “Preview Changes” dialog is focused in Dark/Blend theme

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#11048]
## Details about Problem
VS Version: Main\31519.82
NuGet Version: Dev\6.0.0.155
OS Version: Windows-10-Enterprise-21H1
## Repro steps
1.	Launch VS and set the VS theme to “Dark” (or “Blend” in Blend for VS) through clicking menu “Tools->Options->Environment->Color Theme”
2.	Create a solution [e.g. ASP.NET Web Application (.NET Framework 4.8) ->Single Page Application].
3.	Open the project-level PM UI and go to “Updates” tab.
4.	Check “Select all packages” check-box and click the “Updates” button.
5.	Press “Tab” button on keyboard after the “Preview Changes” dialog prompt.
6.	Observe when the focus is on the package list text-box.
## Expected
Easy to see the package list text-box of “Preview Changes” dialog is focused as below screenshot.
![image](https://user-images.githubusercontent.com/20632625/126312153-376f5e11-c3dc-4ada-b2f6-84202c623825.png)
## Actual
Hard to see the package list text-box of “Preview Changes” dialog is focused (there is no dash-line rectangle show) as below screenshot.
![image](https://user-images.githubusercontent.com/20632625/126312218-bff99346-2587-41a4-b78d-9c662cc31302.png)
## Notes:
1.It repro in both old Dark theme and new Dark theme (experimental Dark theme).
2.It doesn’t repro in Light/Blue/Blue (extra contrast) theme.
3.It is not a regression since it did repro on D16.8\30420.98 with implicit NuGet Client Dev\5.8.0.6776.

**Comment by aortiz-msft** [comment]
@donnie-msft  - Do we use a custom control here?

**Comment by v-luzh** [comment]
It still reproes on VS Main\35222.175 with implicit NuGet Client Dev\6.12.0.82.

**Comment by jgonz120** [comment]
We are using some of the system brushes. I believe these borders should be a 3:1 ratio
